doc/whats_new/v0.23.rst

```diff
@@ -128,6 +128,11 @@ Changelog
   instead of predictions.
   :pr:`14848` by :user:`Venkatachalam N <venkyyuvy>`.
 
+- |API| Deprecated public attributes `standard_coef_`, `standard_intercept_`, 
+  `average_coef_`, `average_intercept_` in :class:`linear_model.BaseSGD`
+  (inherited by :class:`linear_model.SGDRegressor`).
```

Comment:
Let's be exhaustive as well for the impacted public classes. If I did not forget anything (maybe good if someone can double check), it affects ``PassiveAggressiveClassifier``, ``PassiveAggressiveRegressor``, ``SGDClassifier`` and ``SGDRegressor``

Reply:
yes, done